platform/commonUI/edit/bundle.js

```diff
@@ -165,6 +167,15 @@ define([
                     "implementation": SaveAction,
                     "name": "Save",
                     "description": "Save changes made to these objects.",
+                    "depends": [],
+                    "priority": "mandatory"
+                },
+                {
+                    "key": "save-as",
+                    "category": "conclude-editing",
+                    "implementation": SaveAsAction,
+                    "name": "Save As",
```

Comment:
Changed key and name of "save-as" to "save". Seems to work, and applies-to is sufficient to distinguish, as you say.